exporters-yaml/sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: robot
```

Comment:
@ramius345 Should we give the Service Account a meaningful name other than robot?

Reply:
Changed to exporter-robot